QuizApp/App/Networking/Models/CategoryNetworkModel.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-enum CategoryNetworkModel: String {
+enum CategoryNetworkModel: String, Codable {
```

Comment:
Category should have a model for each layer, just like other models. 
You are using network/client layer model for multiple layers.